packages/cloud-cognitive/src/components/ComboButton/ComboButton.js

```diff
@@ -146,15 +122,27 @@ export let ComboButton = React.forwardRef(
 ComboButton = pkg.checkComponentEnabled(ComboButton, componentName);
 ComboButton.displayName = componentName;
 ComboButton.propTypes = {
+  /** Provide an optional flag to disable entire ComboButton */
+  buttonDisabled: bool,
```

Comment:
Do we need three disabled states here? I’m seeing `buttonDisabled`, `disabled`, and `itemDisabled`, but only `disabled` is used in ComboButton. If these are meant for just the stories, they should be set there instead.